tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -427,21 +430,10 @@
       ,
       {
         "name": "lightstep",
-        "http2_protocol_options": {},
-        {{ if .lightstepSecure }}
-        "tls_context": {
-          "common_tls_context": {
-            "alpn_protocols": [
-              "h2"
-            ],
-            "validation_context": {
-              "trusted_ca": {
-                "filename": "{{ .lightstepCacertPath }}"
-              }
-            }
-          }
-        },
+        {{ if .tracing_tls }}
+        "tls_context": {{ .tracing_tls }},
         {{ end }}
+        "http2_protocol_options": {},
```

Comment:
Why is this added? It seems unrelated maybe?

Reply:
This will break http/1.1 servers. There is no negotiation at client level.

Reply:
This already exists for the lightstep block. So this is not a new change